etc/kayobe/ansible/deploy-radosgw-usage-exporter.yml

```diff
@@ -0,0 +1,68 @@
+---
+- name: Deploy RADOS gateway usage exporter
+  hosts: monitoring
+  gather_facts: false
+  vars:
+    venv: "{{ virtualenv_path }}/openstack"
```

Comment:
Why is the venv assume the name `openstack`?

Reply:
Currently it needs to query ec2 credential with openstackclient. I'll try generalise this as it gets developed more. This is still work in progress.